baybe/recommenders/base.py

```diff
@@ -212,15 +194,20 @@ def _select_candidates_and_recommend(
             NotEnoughPointsLeftError: If there are fewer points left for potential
                 recommendation than requested.
         """
+        is_hybrid_space = searchspace.type is SearchSpaceType.HYBRID
+
         # Get discrete candidates
-        _, candidates_comp = subspace_discrete.get_candidates(
-            allow_repeated_recommendations=self.allow_repeated_recommendations,
-            allow_recommending_already_measured=self.allow_recommending_already_measured,
+        _, candidates_comp = searchspace.discrete.get_candidates(
+            allow_repeated_recommendations=True
```

Comment:
According to myGPT, this statement can be simplified as `allow_repeated_recommendations = is_hybrid_space or self.allow_repeated_recommendations`, and I also think that this is indeed equivalent.
However, if you prefer the current version (which is more readable) then feel free to not change this in this function.

Reply:
like it and will change